src/cloud/benchflow/sender/sender.go

```diff
@@ -164,8 +164,10 @@ func consumeFromTopic(t TransformerSetting) {
 				continue
 				}
 			fmt.Println(t.Topic+" received: "+msg.Minio_key)
+			minioKeys := strings.Split(msg.Minio_key, ",")
```

Comment:
You should do this only for the collectors for which it holds. According to the following issue, currently this holds only for the stats collector: https://github.com/benchflow/collectors/issues/46